app/views/alaveteli_pro/info_requests/_info_request.html.erb

```diff
@@ -2,7 +2,9 @@
      class="request request--<%= info_request.state.summarised_phase %>">
   <h3 class="request__title">
     <% if info_request.embargo %>
-      <span class="embargo-indicator embargo-indicator--small">
+      <span class="embargo-indicator embargo-indicator--small"
+                   title="<%= I18n.l(info_request.embargo.publish_at,
+                                    format: '%d %B %Y') %>">
```

Comment:
I think we should give more context here:

```erb
title="<%= _('Embargoed until {{date}}', :date => I18n.l(info_request.embargo.publish_at, format: '%d %B %Y')) %>
```

Also align `title` with `class`.

Reply:
There's also [a helper](https://github.com/mysociety/alaveteli/blob/develop/app/helpers/date_time_helper.rb#L16) that looks like it would be good to use to convert the date.

Reply:
Cool, thanks - just having a look round to see if 'Embargoed until' is the right phrasing as I half remember a conversation that being a potentially confusing term

Reply:
Looks like 'Private until' will fit in better with what we have in the Pro new request form. Will happily correct it later if I got that wrong

Reply:
Nope sounds good to me – have a quick check to see if we already do this anywhere else (I guess search for `publish_at`) but +1 on UI consistency.

Reply:
The wording around making a new request (or viewing an existing one) from the dashboard 'Privacy' / 'private' so that feels like the right thing

Reply:
Hmm, actually we have "This request will be private on {{site_name}} until {{date}}" but that seems too long for rollover text. Easy enough to tweak the working once we have the code so going to leave it as-is